config/redirects

```diff
@@ -43,6 +43,7 @@ raw: docs/drivers/tutorial/getting-started-with-csharp-driver -> http://mongodb.
 raw: docs/drivers/tutorial/serialize-documents-with-the-csharp-driver -> http://mongodb.github.io/mongo-csharp-driver/2.2/getting_started
 raw: docs/drivers/tutorial/use-csharp-driver -> http://mongodb.github.io/mongo-csharp-driver/2.2/getting_started
 raw: docs/drivers/tutorial/use-linq-queries-with-csharp-driver -> http://mongodb.github.io/mongo-csharp-driver/2.2/getting_started
```

Comment:
Q: Did we verify that nothing else needs to be updated here? The new docs have a LINQ page, e.g.